modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc

```diff
@@ -14,16 +14,4 @@ To run a process, you can :
 * Open the process diagram and select the pool, then click *_Run_* in the Cool bar. The process in the selected pool runs. Only one pool can be run at a time.
 
 If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The
-process must be configured for the environment in which it runs.
-
-There are two modes in which you can run a process, standalone and portal:
-
-* Standalone mode uses templates to contain the form application, These templates are editable in Studio. When you run a process from the Studio, it runs in standalone mode.
-* Portal mode uses a different container. When you run a process from the Portal, it runs in portal mode.
-This means that if you include a script in the templates, which are the standalone container, the script cannot be called by the process when it runs in portal mode.
-
-To include a script in a process so it can be used in portal mode, there are the following options:
-
-* In the form, add an HTML widget that contains it``.
-
-After a process is xref:admin-application-process-list.adoc[deployed and initialized] in Bonita Portal, it is run by xref:cases.adoc[starting a case].
+process must be configured for the environment in which it runs.
```

Comment:
If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The process must be configured for the environment in which it runs. =>  A process can only run in the environment(s) in which it has been configured. If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can choose the environment in which to run the process.